packages/2018-housing-affordability/src/components/AffordableRentalUnitsDwindling/index.js

```diff
@@ -1,19 +1,134 @@
 import React from 'react';
+import { connect } from 'react-redux';
+import { css } from 'emotion';
 
-import { CivicStoryCard, Placeholder } from '@hackoregon/component-library';
+import '@hackoregon/component-library/assets/vendor/react-select.min.css';
+
+import { HorizontalBarChart, CivicStoryCard, Dropdown } from '@hackoregon/component-library';
+import { percentage } from '@hackoregon/component-library/src/utils/formatters';
+
+import {
+  fetchAllARUCities,
+  fetchARUCity,
+  setARUCity,
+} from '../../state/affordable-rental-units/actions';
+import {
+  isAllCitiesLoading,
+  isCityDetailLoading,
+  getCityError,
+  getSelectedCity,
+  getSelectedCityData,
+  getSelectedCityLowRank,
+  getSelectedCityHighRank,
+  getAllCities,
+} from '../../state/affordable-rental-units/selectors';
+
+const loader = css`
+  background: #EEE;
+  padding: 30px;
+`;
+
+const error = css`
+  background: #FEE;
+  color: #C00;
+  padding: 30px;
+`;
+
+const inputClass = css`
+  width: 100%;
+`;
+
+const emphasis = css`
+  color: #000;
+`;
+
+const capitalize = str => str.length && str.split(' ')
+  .reduce((full, word) => `${full} ${word[0].toUpperCase() + word.substring(1)}`, '')
+  .trim();
 
```

Comment:
you could also use `titleCase`, which I added to formatters in the component lib!